pkgs/racket-doc/scribblings/more/more.scrbl

```diff
@@ -95,6 +95,8 @@ start @exec{racket} with no command-line arguments:
   > 
 }
 
+@margin-note{Set your @tt{PATH} environment variable so you can use @exec{raco} and other Racket command line functions. On MacOS: @tt{sudo sh -c 'echo "/Applications/Racket v@version{}/bin" >> /etc/paths.d/racket'} (This assumes you have installed Racket in the @tt{Applications} folder). On Windows: add the racket @tt{bin} path to @tt{Path} in 'Environment Variables' (under System Properties, Advanced tab)}
```

Comment:
Should we wrap this?

Reply:
what is the etiquette here? Can I click 'Resolve conversation' or does it need to be a repo owner?

Reply:
![image](https://user-images.githubusercontent.com/108047/83858759-def4ef80-a714-11ea-9949-de571b2e3e0a.png)

Reply:
I think what @pmatos meant is that the Scribble source should be broken up to several lines, not one long line.

Reply:
wrap fixed
![image](https://user-images.githubusercontent.com/108047/83862849-698c1d80-a71a-11ea-9a16-eeedacd3a3f1.png)

still renders OK
![image](https://user-images.githubusercontent.com/108047/83862889-7a3c9380-a71a-11ea-98e2-2fa7fd0fc534.png)